manifests/init.pp

```diff
@@ -136,7 +136,7 @@
   $redis_slave_output_buffer_soft_limit = $redis::params::redis_slave_output_buffer_soft_limit,
   $redis_slave_output_buffer_soft_limit_max_interval = $redis::params::redis_slave_output_buffer_soft_limit_max_interval,
   $redis_snapshotting = $redis::params::redis_snapshotting,
-  $manage_config_file = $redis::params::manage_config_file
```

Comment:
The intent is to update the configs (both init script and redis config) but not restart the service. We're trying to strike a balance between endless drift because nothing gets updated and immediate restart which causes freakouts in our system. I'm amenable to the idea of not updating the config either or simply removing the notify on the init script file resource but the way it is now requires people to turn on manage_config_file to update their configs and then we end up with a bulk restart. This way, the config updates and engineers can go in and restart at their leisure.